tests/integration/customizations/s3/__init__.py

```diff
@@ -25,9 +26,16 @@ def make_s3_files(session, key1='text1.txt', key2='text2.txt'):
     """
     region = 'us-west-2'
     bucket = create_bucket(session)
-    string1 = "This is a test."
-    string2 = "This is another test."
+
+    if size:
+        string1 = "*" * size
+        string2 = string1
+    else:
+        string1 = "This is a test."
+        string2 = "This is another test."
+
     client = session.create_client('s3', region_name=region)
+    client.meta.events.unregister('before-sign.s3', fix_s3_host)
```

Comment:
This is a workaround to the broken pipe issue, where we cannot send significant string bodies.